dev/com.ibm.ws.jdbc/resources/com/ibm/ws/rsadapter/resources/IBMDataStoreAdapterNLS.nlsprops

```diff
@@ -250,6 +250,11 @@ DSRA4005.invalid.logintimeout=DSRA4005E: The server does not support the loginTi
 DSRA4005.invalid.logintimeout.explanation=The loginTimeout property is not supported when it is specified for a DataSource that is backed by a java.sql.Driver implementation because the server is unable to call getLoginTimeout on the java.sql.Driver class.
 DSRA4005.invalid.logintimeout.useraction=Modify the DataSource configuration to use a DataSource implementation instead of a java.sql.Driver implementation, or remove the loginTimeout property.
 
+# {0} the vendor class name, {1} name of the DataSource, 
+DSRA4006.null.connection=DSRA4006E: The java.sql.Driver {0} returned a null connection for {1} DataSource.
```

Comment:
I don't see a need to include "java.sql.Driver" here given that we will already by printing out the vendor impl class name.  Also, the reference to {1} DataSource is awkward.
Recommend:
"Data source {1} is unable to connect to the database because {0} returned a null connection."